lib/matplotlib/__init__.py

```diff
@@ -1529,6 +1529,9 @@ def _replacer(data, key):
     following arguments are replaced by **data[<arg>]**:
 
     {replaced}
+    
+    Objects passed as **data** must support item access (``data[<arg>]``) and 
+    membership test (``<arg> in data``).
```

Comment:
This is inconsistent with the actual test, and with the desired behavior; the structured ndarray does not support the membership test.

Reply:
Maybe should say 'support `<arg> in data` or be a numpy structured array`?